### DISCUSSION TITLE
"Fix Flows" functionality for sf cli plugin

### DISCUSSION THREAD

**Discussion by k-capehart** [Flow-Scanner/lightning-flow-scanner#125] [Ideas]
It would be great if we could have the same "Fix Flows" functionality in the sf cli plugin as we do in the VS Code extension.

I think we could either do it as a flag: `sf flow scan --fix`
Or as a separate command: `sf flow fix`

Anyone have any preferences? 

CC: @junners

**Comment by junners** [comment]
`sf flow fix` sounds better from a taxonomy perspective!

**Comment by junners** [comment]
implemented
